templates/inspec/examples/attributes/google_compute_autoscaler.erb

```diff
@@ -0,0 +1,4 @@
+gcp_project_id = attribute(:gcp_project_id, default: '<%= external_attribute('gcp_project_id') -%>', description: 'The GCP project identifier.')
+gcp_zone = attribute(:gcp_zone, default: '<%= external_attribute('gcp_zone') -%>', description: 'The GCP project zone.')
+instance_group_manager = attribute('instance_group_manager', default: <%= JSON.pretty_generate(grab_attributes['instance_group_manager']) -%>, description: 'Instance group manager definition')
```

Comment:
Out of curiosity, what's the difference between calling `pretty_generate` and `.to_json`?

Reply:
`pretty_generate` adds new lines and spaces and formats the JSON in a (mostly) human readable way. `to_json` just dumps it all on one line

Reply:
And we're relying on the fact that JSON.pretty_generate outputs something that happens to look like native Ruby code, right?

(Not a problem with that)

Reply:
I guess we are depending on the fact that Ruby can parse JSON into a hash, regardless of the spacing in that JSON